features/custom_error_pages.feature

```diff
@@ -17,6 +17,12 @@ Feature: I want to make error pages follow the general design of the site.
     And the response status should be "404"
     And I should see "We're sorry, but we couldn't find the page you requested"
 
+   Scenario: 404 page when opening an invalid opportunity
+    When I visit "/volunteer_ops/foo-bar-op"
+    And the page should be titled "404 Page Not Found"
+    And the response status should be "404"
+    And I should see "We're sorry, but we couldn't find the page you requested"
+    
```

Comment:
Added scenario to test invalid opportunity, similar to previous scenario which test for 404 page when invalid project (i.e. organisation)